package.json

```diff
@@ -165,10 +165,20 @@
                 "scopeName": "text.css.hubl",
                 "path": "./syntaxes/css-hubl.json"
             }
-        ]
+        ],
+        "configuration": {
+            "title": "HubL",
```

Comment:
This "title" is the main title that shows at the top of settings, right? If so, should we rename it to match the name of our extension? I looked around a little bit and that seemed to be a fairly common pattern.

Reply:
Good point. Since the name of the extension is: "HubSpot VS Code Extension", I updated this to be "HubSpot", to be consistent with other extensions:
![image](https://user-images.githubusercontent.com/9009552/114778268-a06f8e80-9d42-11eb-96bb-baceac2232c8.png)